include/component/switch.hh

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include "core/util.hh"
+#include "core/parameter.hh"
+
+// instead of using cv.hh, we directly use juce::CachedValue<bool>
+
+namespace box{
+struct Switch{
+    juce::CachedValue<bool> &value_;
+    uint8_t x_, y_;
+    Color colour_;
+    std::string name_;
+
+    Switch(juce::CachedValue<bool> &value, uint8_t x, uint8_t y, std::string name="")
+    : value_{value}, x_{x}, y_{y}, name_{name}, colour_{RED}
```

Comment:
Why don't you have a `color` parameter instead of making it always red?
See:
https://github.com/wang-edward/box/blob/72db56ec4cd4a7ce7df8b8ad735cd80442310427/include/component/knob.hh#L16C64-L16C75